compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/textfield/KeyboardActionsExample.kt

```diff
@@ -109,7 +109,7 @@ private fun TextBlock(
         OutlinedTextField(
             value = textState.value,
             onValueChange = { textState.value = it },
-            modifier = Modifier.fillMaxWidth().padding(top = 8.dp).onKeyEvent { println("KEY EVENT $this"); false },
+            modifier = Modifier.fillMaxWidth().padding(top = 8.dp),
```

Comment:
was it necessary to remove `onKeyEvent { ... }` here?

Reply:
it was our sanity from ancient times but having a dedicated test is a better option